File: 은지/LV1/시저 암호.py
def solution(s, n):
    answer = ''
    for x in s:
        # 공백인 경우
        if x == ' ':
            answer += ' '
        # 다시 돌아가야하는 경우
        # 1. 대문자
        elif 65 <= ord(x) <= 90 and ord(x) + n > 90:
            answer += chr(ord(x) + n - 26)
        # 2. 소문자
        elif 97 <= ord(x) <= 122 and ord(x) + n > 122:
            answer += chr(ord(x) + n - 26)
        # 돌아갈 필요없는 경우
        else:
            answer += chr(ord(x) + n)
    return answer

File: 준영/프로그래머스/LV1/시저 암호.js
function solution(s, n) {
  var answer = ''
  
  for (const i of s) {
    if (i === ' ') {
      answer += ' '
      continue
    }

    var x = i.charCodeAt()

    if (x >= 97 && x <= 122) {
      var to_charCode = (x - 'a'.charCodeAt() + n) % 26
      to_charCode += 'a'.charCodeAt()
      answer += String.fromCharCode(to_charCode)
    }
    
    if (x >= 65 && x <= 90) {
      var to_charCode = (x - 'A'.charCodeAt() + n) % 26
      to_charCode += 'A'.charCodeAt()
      answer += String.fromCharCode(to_charCode)
    }

  }

  return answer
}

solution('Z', 1)